### ISSUE TITLE
Identity scaffolder --listFiles option doesn't work properly if default UI was already overridden

### ISSUE THREAD

**Issue by dot-sent** [aspnet/Scaffolding#972]
Environment: Ubuntu Linux 16.04.6 LTS, dotnet info:

```
.NET Core SDK (reflecting any global.json):
 Version:   2.2.105
 Commit:    7cecb35b92

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  16.04
 OS Platform: Linux
 RID:         ubuntu.16.04-x64
 Base Path:   /usr/share/dotnet/sdk/2.2.105/

Host (useful for support):
  Version: 2.2.3
  Commit:  6b8ad509b6

.NET Core SDKs installed:
  2.2.105 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.9 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.3 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.9 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.3 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.6 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.9 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.3 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
```

## Steps to reproduce:
```

dotnet new mvc
dotnet add package Microsoft.VisualStudio.Web.CodeGeneration.Design
dotnet aspnet-codegenerator identity -fi "Account.Login;Account.AccessDenied"
dotnet aspnet-codegenerator identity -lf
```

It's a simplified example; in real-world project it might be needed to scaffold another file after the initial ones are already scaffolded or overridden and one might need to consult a list of files quickly.

## Expected behavior:

Scaffolder gives a list of files that it can generate.

## Actual behavior:

```
dot_sent@sergey-pc:~/Projects/test$ dotnet aspnet-codegenerator identity -lf
Building project ...
Finding the generator 'identity'...
Running the generator 'identity'...
The below files exist. Use '--force' to overwrite: Areas/Identity/Data/testIdentityDbContext.cs
Areas/Identity/Pages/Account/AccessDenied.cshtml
Areas/Identity/Pages/Account/AccessDenied.cshtml.cs
Areas/Identity/Pages/Account/Login.cshtml
Areas/Identity/Pages/Account/Login.cshtml.cs
   at Microsoft.VisualStudio.Web.CodeGeneration.ActionInvoker.<BuildCommandLine>b__6_0()
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at Microsoft.VisualStudio.Web.CodeGeneration.ActionInvoker.Execute(String[] args)
   at Microsoft.VisualStudio.Web.CodeGeneration.CodeGenCommand.Execute(String[] args)
RunTime 00:00:04.08
dot_sent@sergey-pc:~/Projects/test$ 

```

If I add the `-f` parameter, the scaffolder lists files successfully.

### Target framework(s):

`netcoreapp2.2`

### Package version of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All (if applicable):

`2.2.3` - both

### Package version of Microsoft.VisualStudio.Web.CodeGeneration.Design - this may be added to your project by scaffolding:

`2.2.3`

**Comment by CamSoper** [comment]
@dot-sent My friend @scottaddie  and I just hit this issue in 2.2, but it looks like it's fixed in 3.0. 

BTW, what a really well-written issue! Kudos!

**Comment by dot-sent** [comment]
Thanks @CamSoper - I try to report any issues in a way I would like to get some reported to me from my users :)

If this is fixed in 3.0, should this issue be closed then?

**Comment by CamSoper** [comment]
@dot-sent Let's keep it open for the moment -- It *seems* fixed in 3.0, but there are a bunch of issues around the identity scaffold in general that could be masking the issue (to repro in 3.0, you need to add a bunch of packages manually and it still doesn't leave you in a usable state).

**Comment by vijayrkn** [comment]
@dot-sent - Can you please check with .NET Core 3.0 & see if this issue is fixed? If so, can you close this issue here?

& Thanks for such a detailed issue description.

**Comment by vijayrkn** [comment]
Closing since the issue is fixed in 3.0
